index.md

```diff
@@ -43,8 +43,10 @@ static linking support, and clean code.
 
 - [Comparison] with other Linux libc implementations
 - [Install notes], [readme], and [release notes] in the official repo.
-- [Generating cross compilers for musl][generating-cross] - also features
-  [pre-built cross compiler downloads][pre-built-cross] for different arches
+- [Musl cross][musl-cross] - This is a small suite of scripts and patches to build a musl libc cross-compiler.
+- [Musl cross make][musl-cross-make] - Second genratiion advanced makefile-based approach for producing musl-targeting cross compilers
```

Comment:
genratiion ?

Reply:
put mcm first and mcm second. add "Official&Recommended." to mcm.

Reply:
mcm *is* the official way to build a musl-targeting toolchain, so it should be marked as such. even though musl-cross is maintained, i'd prefer users using mcm, if possible. musl-cross only exists still because it supports older compiler versions.